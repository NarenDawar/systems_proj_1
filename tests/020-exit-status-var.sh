#
# Test that minibash correctly sets the $? variable to the exit code of the last program.
#

echo Hello World 123
echo $?
false
echo $?
true
echo $?
