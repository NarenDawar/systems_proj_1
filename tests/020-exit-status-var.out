Hello World 123
0
1
0
